frontend-web/src/app/components/ui/Navbar.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+
+import { motion } from 'framer-motion';
+import { useTranslations } from 'next-intl';
+import React, { JSX, useCallback, useEffect, useState } from 'react';
+
+export default function Navbar(): JSX.Element {
+  const t = useTranslations('navbar'); // Using translations for the navbar
+  const [activeSection, setActiveSection] = useState<string | null>(null);
+
+  const sections = [
+    { id: 'header', label: t('header') },
```

Comment:
utiliser des constantes à la place des id en dur.
```suggestion
    { id: ANCHOR.HEADER, label: t(ANCHOR.HEADER) },
```

et ANCHOR.HEADER est un des champs d'un enum